apispec/registers-delta-spec.yml

```diff
@@ -0,0 +1,66 @@
+post:
+  summary: Accepts an incoming Register delta, transforms it into an avro schema and puts it onto a Kafka topic.
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/RegisterDelta'
+  responses:
+    '200':
+      description: Successfully produced message onto Kafka topic.
+    '400':
+      description: Bad request body - validation errors.
+    '401':
+      description: Unauthorised - missing api key in header.
+    '500':
+      description: Internal server error has occured.
+
+components:
+  schemas:
+    RegisterDelta:
+      type: object
+      properties:
+        company_number:
+          type: string
+        directors:
+          $ref: '#/components/schemas/RegisterSection'
+        llp_members:
+          $ref: '#/components/schemas/RegisterSection'
+        llp_usual_residential_address:
+          $ref: '#/components/schemas/RegisterSection'
+        members:
+          $ref: '#/components/schemas/RegisterSection'
+        persons_with_significant_control:
+          $ref: '#/components/schemas/RegisterSection'
+        secretaries:
+          $ref: '#/components/schemas/RegisterSection'
+        usual_residential_address:
+          $ref: '#/components/schemas/RegisterSection'
+        delta_at:
+          type: string
+        CreatedTime:
```

Comment:
Is this supposed to be snake case?

Reply:
originally I had it snake case but changed it to match the same field in several other delta specs for consistency - psc statement, disqualification, officer, company and psc deltas all have it this way